ci_scripts/remote_python_packaging.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash 
 
-export BRANCH=${CIRCLE_BRANCH:?branch to build}
-export REPONAME=${CIRCLE_PROJECT_REPONAME:?repository name}
+export BRANCH=${CIRCLE_BRANCH:master}
```

Comment:
i do not know much about CIRCLE, but the idea is to use a default if `CIRCLE_BRANCH` is under?

if so, it must probably read

```suggestion
export BRANCH=${CIRCLE_BRANCH:-master}
```

found that at https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html#Shell-Parameter-Expansion